### PR TITLE
MODCAL-68 drop deprecated functions

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Purpose
Create sql script to remove deprecated functions.

Resolves: [MODCAL-68](https://issues.folio.org/browse/MODCAL-68)

## Links
https://issues.folio.org/browse/MODCAL-68

### CODE REVIEW

File: src/main/resources/templates/db_scripts/uuidFunctions.sql [hunk 1]
```diff
@@ -3,3 +3,9 @@ BEGIN
 <#include "uuid.ftl">
 EXCEPTION WHEN OTHERS THEN NULL;
 END $uuidftl$;
+
+DO $$ BEGIN DROP FUNCTION count_estimate_smart; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION count_estimate_smart_depricated; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION set_id_injson_actual_opening_hours; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION set_id_injson_exceptional_hours; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION set_id_injson_regular_hours; EXCEPTION WHEN OTHERS THEN END; $$;
```
Comment (julianladisch) [inline src/main/resources/templates/db_scripts/uuidFunctions.sql:11]: When testing the upgrade as explained in
https://issues.folio.org/browse/MODCAL-68?focusedCommentId=74254&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-74254
this doesn't work.
See #117 how to fix this trigger issue.
These functions are not related to uuid.ftl.
Can you move all *.sql scripts that do the upgrade from RMB 24 to RMB 29 into one file? See #117 for an example.

**Comment by julianladisch** [comment]
mod-calendar gives this warning when running:
```
07 Apr 2020 15:06:07:893 WARN  RamlsAPI [15937eqId] Unable to get RAMLs list from /home/julian/git/mod-calendar/ramls/raml.list
java.io.UncheckedIOException: /home/julian/git/mod-calendar/ramls/raml.list: Resource not found
        at org.folio.util.ResourceUtil.asString(ResourceUtil.java:63) ~[util-29.4.0.jar:?]
        at org.folio.util.ResourceUtil.asString(ResourceUtil.java:28) ~[util-29.4.0.jar:?]
        at org.folio.rest.impl.RamlsAPI.getRamlsList(RamlsAPI.java:85) ~[domain-models-runtime-29.4.0.jar:?]
        at org.folio.rest.impl.RamlsAPI.<clinit>(RamlsAPI.java:37) ~[domain-models-runtime-29.4.0.jar:?]
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_242]
        at java.lang.Class.forName(Class.java:264) ~[?:1.8.0_242]
        at org.folio.rest.tools.utils.InterfaceToImpl.convert2Impl(InterfaceToImpl.java:54) ~[domain-models-runtime-29.4.0.jar:?]
        at org.folio.rest.RestVerticle.runHook(RestVerticle.java:1025) ~[domain-models-runtime-29.4.0.jar:?]
        at org.folio.rest.RestVerticle.start(RestVerticle.java:270) ~[domain-models-runtime-29.4.0.jar:?]
        at io.vertx.core.Verticle.start(Verticle.java:66) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$9(DeploymentManager.java:556) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:510) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:518) [netty-transport-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor$6.run(SingleThreadEventExecutor.java:1044) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
Caused by: java.io.FileNotFoundException: Resource not found
        at org.folio.util.ResourceUtil.asString(ResourceUtil.java:58) ~[util-29.4.0.jar:?]
        ... 19 more
07 Apr 2020 15:06:07:904 WARN  JsonSchemasAPI [15948eqId] Unable to get JSON Schemas list!
java.io.UncheckedIOException: /home/julian/git/mod-calendar/ramls/json-schema.list: Resource not found
        at org.folio.util.ResourceUtil.asString(ResourceUtil.java:63) ~[util-29.4.0.jar:?]
        at org.folio.util.ResourceUtil.asString(ResourceUtil.java:28) ~[util-29.4.0.jar:?]
        at org.folio.rest.impl.JsonSchemasAPI.getJsonSchemasList(JsonSchemasAPI.java:90) ~[domain-models-runtime-29.4.0.jar:?]
        at org.folio.rest.impl.JsonSchemasAPI.<clinit>(JsonSchemasAPI.java:40) ~[domain-models-runtime-29.4.0.jar:?]
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_242]
        at java.lang.Class.forName(Class.java:264) ~[?:1.8.0_242]
        at org.folio.rest.tools.utils.InterfaceToImpl.convert2Impl(InterfaceToImpl.java:54) ~[domain-models-runtime-29.4.0.jar:?]
        at org.folio.rest.RestVerticle.runHook(RestVerticle.java:1025) ~[domain-models-runtime-29.4.0.jar:?]
        at org.folio.rest.RestVerticle.start(RestVerticle.java:270) ~[domain-models-runtime-29.4.0.jar:?]
        at io.vertx.core.Verticle.start(Verticle.java:66) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$9(DeploymentManager.java:556) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38) ~[vertx-core-3.8.4.jar:3.8.4]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:510) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:518) [netty-transport-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor$6.run(SingleThreadEventExecutor.java:1044) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.42.Final.jar:4.1.42.Final]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
Caused by: java.io.FileNotFoundException: Resource not found
        at org.folio.util.ResourceUtil.asString(ResourceUtil.java:58) ~[util-29.4.0.jar:?]
        ... 19 more
```
And it doesn't return anything when calling
`curl -H "X-Okapi-Tenant: diku" http://localhost:8081/_/jsonSchemas`:
```
[ "" ]
```

File: src/main/resources/templates/db_scripts/uuidFunctions.sql (hunk 1 above)
Comment (roman-barannyk) [reply src/main/resources/templates/db_scripts/uuidFunctions.sql]: @julianladisch  Thanks!

**Comment by roman-barannyk** [comment]
It seems like this warning is not related with my changes. Can we fix it in scope of another story?

**Comment by julianladisch** [comment]
Yes, I approve this pull request.

**Pull request merged**